### ISSUE TITLE
Clipboard text ignores new line symbols

### ISSUE THREAD

**Issue by bereg2k** [selenide/selenide-selenoid#20]
Consider the following test:

```java
@Test
public void textAreaTest() {
    open("https://www.javatpoint.com/html-textarea");
    
    var lines = "1st line\n" + "2nd line\n" + "3rd line";
    clipboard().setText(""); // clear the clipboard if it's not empty
    
    var textArea = $("p + textarea");
    textArea.setValue(lines);
    textArea.sendKeys(chord(CONTROL, "a"));
    textArea.sendKeys(chord(CONTROL, "c"));
    
    clipboard().shouldHave(content(lines));
}
```

This test is **PASSED** locally, but is **FAILED** in Selenoid:
```
clipboard should have content '1st line
2nd line
3rd line'
Actual value: 1st line2nd line3rd line
Timeout: 4 s.
```

The issue is with incorrect processing of the result here `org.selenide.selenoid.SelenoidClient#getClipboardText`:
```java
try (BufferedReader reader = new BufferedReader(new InputStreamReader(connection.getInputStream()))) {
    return reader.lines().collect(Collectors.joining());
}
```

`Collectors.joining()` simply joins all the lines that are actually separated by new line symbols (`\n`) ignoring them in the final result. This approach only works for copying single-line inputs. I think it will be better if the method `getClipboardText()` actually returned the copied text as-is, with line breaks.

**Comment by asolntsev** [comment]
fixed in 2.2.4

**Comment by bereg2k** [comment]
Awesome, thanks!
